app/assets/javascripts/fe/fe.public.nojquery.js

```diff
@@ -43,9 +43,26 @@
   });
 
   window.fe = {};
-  window.fe.pageHandler = {
+  fe.pageHandler = {
     initialize : function(page) {
-      this.auto_save_frequency = 30;  // seconds
+      this.afe = {};
+      fe.pageHandler = {
```

Comment:
This looks a bit odd with `fe.pageHandler` being defined inside of the initialize function of the definition of `fe.pageHandler`